katalon-start-page/marketing-content.md

```diff
@@ -1,9 +1,7 @@
-### [Get to win $100 by helping us improve Katalon Studio](https://forum.katalon.com/t/survey-get-to-win-100-by-helping-us-improve-katalon-studio/136370?u=albert.vu)
+### [Celebrate our newest Katalon Academy milestones & win $30!](https://forum.katalon.com/t/celebrate-our-newest-katalon-academy-milestones-win-30/141017?u=albert.vu)
 
-[<img src="https://github.com/katalon-studio/katalon-studio/assets/87002543/2e5dcaba-ee32-443b-9b52-4900194a12fe">](https://forum.katalon.com/t/survey-get-to-win-100-by-helping-us-improve-katalon-studio/136370?u=albert.vu)
+[<img src="https://github.com/user-attachments/assets/40758ccd-f3b9-4ad1-a8c8-24511596cc03">](https://forum.katalon.com/t/celebrate-our-newest-katalon-academy-milestones-win-30/141017?u=albert.vu)
```

Comment:
the image is kinda blurred, pls check em
![](https://files.slack.com/files-pri/T1X9F6G2Y-F07H8V6TWCW/screenshot_2024-08-19_at_2.04.38_pm.png)